Makefile

```diff
@@ -73,7 +73,7 @@ coverage: build
 # Builds minio locally.
 build: checks
 	@echo "Building minio binary to './minio'"
-	@CGO_ENABLED=0 go build -tags kqueue --ldflags $(BUILD_LDFLAGS) -o $(PWD)/minio
+	@GOFLAGS="" CGO_ENABLED=0 go build -tags kqueue --ldflags $(BUILD_LDFLAGS) -o $(PWD)/minio
```

Comment:
In case wondering this is done to ensure that we compile with CGO_ENABLED=0 otherwise with GOFLAGS set to '-race' will not allow non-cgo based builds. Also because GOFLAGS is a Go standard env variable to pass in default flags. We just explicitly turn off all flags for `go build`